examples/chip-tool-webos/README.md

```diff
@@ -0,0 +1,143 @@
+# Guide for building Matter example for webOS OSE
+
+## Purpose
+
+This doc provides the build and test guide of Matter client example for webOS
+OSE.
+
+## Tested on:
+
+-   [webOS OSE 2.0](https://www.webosose.org/)
+
+## Prerequisite
+
+Before building and testing Matter example for webOS, you need following items.
+
+-   Ubuntu 18.04 machine (used for building webOS OSE)
+-   Network router
+-   2 raspberrypi4 H/W (First rpi4 used for controller, and second used for
+    controlee)
+-   Flash webOS OSE image on the first raspberrypi4 referring to Step 1 of
+    "Build Instructions" below.
+-   Flash Ubuntu 20.04 or higher on the second raspberrypi4 by referring
+    https://ubuntu.com/download/raspberry-pi
+-   Then, checkout the project-chip/connectedhomeip and build all-clusters-app
+    on examples/all-clusters-app.
+
+## Definition
+
+WEBOS_BUILD_ROOT:
+
+-   Build root path of your webOS OSE build set up on your build machine. ex)
+    /home/cabin15/build-webos
+
+CHIP_ROOT:
+
+-   Root path of the connectedhomeip repository. ex)
+    /home/cabin15/connectedhomeip
+
+IP_ADDRESS_OF_YOUR_RASPBERRYPI4:
+
+-   IP address of other raspberrypi device connected on your router. ex)
+    192.168.0.120
+
+## Build Instructions
+
+Step 1: Build and Flash the webOS OSE
```

Comment:
For all other matter platforms we generally provide the ability to compile from a docker image, including allowing the use of `chip-build-vscode` image to compile any build (including arm64 cross compile).

Is this something that could be done for webOS? as a developer I would find it quite complex to go through all the steps below to just validate that my changes keep compiling on webOS.

Reply:
Dear Andrei,
I appreciate your kind feedback.
Let me check your opinion and get back to you.
Thank you.

Reply:
Dear Andrei,
To follow your guide, I made a script (scripts/example/webos_example.sh) to build chip-tool for webos using webos ndk. In this way, developer who wants to implement their application on webOS OSE could easily work on it. I will modify the README soon. Until then, could you plz review the rest of the changes? Thank you in advance.